commander/src/bootstrapping/templates/lisk-template-ts/templates/module/src/app/modules/module.ts

```diff
@@ -46,9 +53,9 @@ export class <%= moduleClass %> extends BaseModule {
     ];
     public id = <%= moduleID %>;
 
-    public constructor(genesisConfig: GenesisConfig) {
-        super(genesisConfig);
-    }
+    // public constructor(genesisConfig: GenesisConfig) {
```

Comment:
why commented?

Reply:
In bootstrapped application its cause linting errors, so why kept it commented as it would be useful for end user.